aws/cluster.tf

```diff
@@ -82,18 +84,18 @@ resource "aws_security_group" "node_sec_group" {
 
   # HTTP access to the RPC port
   ingress {
-    from_port   = 33145
-    to_port     = 33145
-    protocol    = "tcp"
-    self        = true
+    from_port = 33145
+    to_port   = 33145
+    protocol  = "tcp"
+    self      = true
   }
 
   # HTTP access to the Admin port
   ingress {
     from_port   = 9644
     to_port     = 9644
     protocol    = "tcp"
-    self        = true
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
This opens port 9644 to external traffic, which was causing an issue @r-vasquez and I saw while testing Tristan's PR earlier.